### PR TITLE
D8NID-882 featured news block

### PR DESCRIPTION
**Pull request by johangant**

### CODE REVIEW

File: templates/block/block--views-block--news-featured-news-block.html.twig [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ */
+#}
+{%
+  set classes = [
+  'sub-menu',
```
Comment (omahm) [inline templates/block/block--views-block--news-featured-news-block.html.twig:30]: Check https://digitaldevelopment.atlassian.net/browse/D8NID-888 and see if that applies here.

**Review by omahm — COMMENTED** [review]
Check that ticket I worked on regarding the use of sub-menu class and see if that'll apply here. Looks good tho!

File: templates/block/block--views-block--news-featured-news-block.html.twig (hunk 1 above)
Comment (johangant) [reply templates/block/block--views-block--news-featured-news-block.html.twig:30]: Good spot. Removing the sub-menu class doesn't seem to quite be enough: it needs some padding in there too. I'll merge in the changes from the other PR and see if that helps, or can hold off until that's ready and then apply it to here.

**Comment by johangant** [comment]
> Check that ticket I worked on regarding the use of sub-menu class and see if that'll apply here. Looks good tho!

The sub-menu class here is needed because they're visually distinct from the blocks on the front page. Which is just as well, because when you try to make these use the same classes as the front page blocks then it gets messy pretty quickly! :)

**Pull request merged**